jlm/tooling/CommandLine.cpp

```diff
@@ -991,6 +1033,11 @@ JlmOptCommandLineParser::ParseCommandLineArguments(int argc, char ** argv)
 
   CommandLineOptions_ = JlmOptCommandLineOptions::Create(
       std::move(inputFilePath),
+#ifdef ENABLE_MLIR
+      inputFormat,
+#else
+      JlmOptCommandLineOptions::InputFormat::Llvm,
+#endif
```

Comment:
This is not great. Remove the if-then-else here and hand in the inputFormat. Handle it by defining a variable and defaulting it to Llvm format above such that you only need a single if-then-else. I really would not like to see those all over the place.

Reply:
inputFormat is now always defined and the ifdef is removed.